### ISSUE TITLE
Monkey testing tries to establisch DHCPv6 session though disabled in configuration

### ISSUE THREAD

**Issue by detzen** [rtbrick/bngblaster#196]
**Describe the bug**

I have a simple IPoE DHCPv4 setup (see below) where BNG blaster is the client. BNG is a Juniper MX router. I start my test with `monkey-autostart": false`. All DHCPv4 sessions are established successfully. After starting monkey-testing through an API call, I see that bngblaster now tries to establish DHCPv6 sessions although they are completely disabled in the configuration.

**To Reproduce**
Run bgngblaster with attached configuration and command `bngblaster -C dhcp.json -L test.log -l ip -J report.json -j sessions -j streams -P test.pcap -S run.sock -I`. After starting monkey-testing by API with `bngblaster-cli run.sock monkey-start`, you'll see that bngblaster tries to establish DHCPv6 sessions.

Version (`bngblaster -v`):

```
Version: 0.8.21
Compiler: GNU (11.3.0)
IO Modes: packet_mmap_raw (default), packet_mmap, raw
```

JSON configuration:
```
{
    "interfaces": {
        "access": [
            {
                "__comment__": "DHCP VLAN 132",
                "interface": "eno2",
                "monkey": true,
                "type": "ipoe",
                "ipv6": false,
                "dhcpv6": false,
                "outer-vlan": 2000,
                "inner-vlan": 132,
                "vlan-mode": "N:1"
            },
            {
                "__comment__": "DHCP VLAN 232",
                "interface": "eno2",
                "monkey": true,
                "type": "ipoe",
                "ipv6": false,
                "dhcpv6": false,
                "outer-vlan": 2000,
                "inner-vlan": 232,
                "vlan-mode": "N:1"
            },
            {
                "__comment__": "DHCP VLAN 432",
                "interface": "eno2",
                "monkey": true,
                "type": "ipoe",
                "ipv6": false,
                "dhcpv6": false,
                "outer-vlan": 2000,
                "inner-vlan": 432,
                "vlan-mode": "N:1"
            }
        ]
    },
    "access-line": {
        "agent-remote-id": "DEU.TESTLAB.{session-global}",
        "agent-circuit-id": "bngblaster ens2:{session-global}"
    },
    "dhcp": {
        "enable": true,
        "broadcast": false
    },
    "dhcpv6": {
        "enable": false
    },
    "sessions": {
        "count": 150,
        "reconnect": true,
        "autostart": true,
        "monkey-autostart": false
    }
}

```

1. start bgngblaster with attached configuration
   `bngblaster -C dhcp.json -L test.log -l ip -J report.json -j sessions -j streams -P test.pcap -S run.sock -I`
2. start monky-testing by API
    `bngblaster-cli run.sock monkey-start`
4. watch User Interface

**Expected behavior**

Do to deactivation of IPv6 in the configuration, I would expect, that bngblaster doesn't try to setup IPv6 DHCP sessions at all.

![image](https://github.com/rtbrick/bngblaster/assets/34800197/de24524f-46ec-4ea9-8e11-a7819667f9e5)

**Comment by GIC-de** [comment]
Sure, I will try to release a new version by end of this week and will try to include this fix here as well.

**Comment by detzen** [comment]
That was fast ;-)
Many thanks.
Regards,
Alex

**Comment by GIC-de** [comment]
Fixed with 0.8.22 (https://github.com/rtbrick/bngblaster/releases/tag/0.8.22)!
